# Generated by Django 5.1.6 on 2025-02-25 09:46

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0006_alter_foodgramuser_first_name_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='foodgramuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, max_length=150, unique=True, validators=[django.core.validators.MaxLengthValidator(150)]),
        ),
    ]
